infra-as-code/bicep/modules/policy/assignments/alzDefaults/policyExemptions.bicep

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+/*
+  SUMMARY : Creates a Policy Exemption for a Policy Assignment in a Management Group
+  AUTHOR/S: Cloud for Sovereignty
+*/
+targetScope = 'managementGroup'
+
+@description('Policy Assignment Name')
+param parPolicyAssignmentName string
+
+@description('Policy Assignment Scope Name')
+param parPolicyAssignmentScopeName string
+
+@description('SLZ Policy Set Assignment id')
+param parPolicyAssignmentId string = '/providers/microsoft.management/managementgroups/${parPolicyAssignmentScopeName}/providers/microsoft.authorization/policyassignments/${parPolicyAssignmentName}'
```

Comment:
Rather than generating the resource ID string, we should update the policy assignments module to pass the resource ID as an output and use that instead.

Reply:
Done